spec/ngx_helper.lua

```diff
@@ -51,6 +59,19 @@ local function reset_ngx_state()
   reset_ngx_shared(ngx_shared_original)
 end
 
+local function verify()
```

Comment:
This needs a comment. You could reuse the one you added in the commit.
I think it's important to explain both what this is doing and why.